infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -504,6 +504,151 @@ all the operations for the connector:
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_ID` with `App key` value.
   - `PSOXY_DROPBOX_BUSINESS_CLIENT_SECRET` with `App secret` value.
 
+EOT
+    },
+    jira-server = {
+      source_kind : "jira-server"
+      worklytics_connector_id : "jira-server-psoxy"
+      target_host : var.jira_server_url
+      source_auth_strategy : "oauth2_access_token"
+      display_name : "Jira REST API"
+      identifier_scope_id : "jira"
+      worklytics_connector_name : "Jira Server REST API via Psoxy"
+      secured_variables : [
+        { name : "ACCESS_TOKEN", writable : false },
```

Comment:
Jira Server works with a personal access token; it could work with [certificate](https://developer.atlassian.com/server/jira/platform/oauth/#step-3--authorize) or [user credentials](https://developer.atlassian.com/server/jira/platform/basic-authentication/) but it is not an OAuth2.0 compliant.
For this, I've followed the access token, as it seems less hard to setup and better than providing user/name to the requests.

Reply:
certainly better than username + password, but is it tied to a single user?